### DISCUSSION TITLE
chatglm3 lora微调后output缺少内容

### DISCUSSION THREAD

**Discussion by chaozzzzzzz** [zai-org/ChatGLM3#1132] [Q&A] [answered]
按照在2000条保存提示没有配置文件
![1](https://github.com/THUDM/ChatGLM3/assets/166978925/f7d804fe-de76-4123-a251-e50bbd4ef14b)
output里只含有
![image](https://github.com/THUDM/ChatGLM3/assets/166978925/556d6c34-0b9a-4a90-9846-a828566e3d17)

**Comment by chenchen333-dev** [comment]
> 按照在2000条保存提示没有配置文件 ![1](https://private-user-images.githubusercontent.com/166978925/322251960-f7d804fe-de76-4123-a251-e50bbd4ef14b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxNDQyODYsIm5iZiI6MTcxMzE0Mzk4NiwicGF0aCI6Ii8xNjY5Nzg5MjUvMzIyMjUxOTYwLWY3ZDgwNGZlLWRlNzYtNDEyMy1hMjUxLWU1MGJiZDRlZjE0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxNVQwMTE5NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01Mzc3YjY2ZDEzMjAwZWY1NmRjMTU0M2M3MjU3OWY1NTY2ZDY0MzgyMzNjMmI0ZGJjNmQxMWFhZTQ5Mjk0OGYzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.nFHdmxPOiGDApXg43JwHO1fcBRw70yp5ourAIZWr8S8) output里只含有 ![image](https://private-user-images.githubusercontent.com/166978925/322251988-556d6c34-0b9a-4a90-9846-a828566e3d17.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxNDQyODYsIm5iZiI6MTcxMzE0Mzk4NiwicGF0aCI6Ii8xNjY5Nzg5MjUvMzIyMjUxOTg4LTU1NmQ2YzM0LTBiOWEtNGE5MC05ODQ2LWE4Mjg1NjZlM2QxNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxNVQwMTE5NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01NTI0ZDk2NDE1Y2E0ZTMxYzE0MjI0YTMyYWI3ZmYyOTE3MmViNTI2OWY5ZGE4NzkxZmNmMzhhY2E1YzlkYjE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.QI74Cm9HxwVaNd2JqYb_ACqu6S16wnYVybgjo3UZIxY)

请问你微调时有遇到进程被挂起，无法训练的问题吗(能成功加载模型和打印首条训练数据)
![92b6d233337b7fdb048e77c5717494a](https://github.com/THUDM/ChatGLM3/assets/73687307/60ccf0cd-0dbe-4978-bd2c-72150528f540)

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
推理部分请参考infer进行推理。
关于训练卡住大概率是设备问题，没有遇到过。
